src/PerfView/HeapView/HeapAnalyzer.cs

```diff
@@ -933,12 +932,11 @@ public bool LoadEvents(int procID, int sampleInterval100ns)
             }
 
             // Get the process we want
-            if (!m_processLookup.ContainsKey(procID))
+            if (!m_processLookup.TryGetValue(procID, out m_process))
```

Comment:
📝 This one appears to be a behavior change if `m_process` is not null when this line is reached.